tools/internal_ci/linux/grpc_full_performance_release.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -ex
+
+# Enter the gRPC repo root
+cd $(dirname $0)/../../..
+
+source tools/internal_ci/helper_scripts/prepare_build_linux_perf_multilang_rc
+
+# run 8core client vs 8core server
+tools/run_tests/run_performance_tests.py \
```

Comment:
Actually, this is important:
looks like in the kokoro job configuration, the "driver" machine is acquired from a regular pool of workers.
Here's the problem:
- if you have 2 jobs running at the same time (will happen for sure once you have 2 jobs - full_performance_master and full_performance_releases), they will  fight over the client and server worker VMs.
- so far that hasn't happened because we were "lucky" and didn't run the "full performance" jobs concurrently with anything.

The remedy for this is to setup a special kokoro pool, that only has 1 VM in it.   Then all the jobs that want to access the same set of workers (server, client and client2)  will  use that pool - which will make the job runs exclusive and they will be sharing the resources correctly.

Reply:
Yup, a separate pool should be made for the drivers with a limit of only 1 active workers. I was considering adding a 2nd set of server/clients, but I don't think we not data being generated that frequently. We'll need a 2nd set if we decide to add the full sweep benchmark (Craig has said that it hasn't been used in months and will likely stay unused for the foreseeable future).